src/components/movieForm/index.tsx

```diff
@@ -12,12 +12,42 @@ import useStyles from './styles';
 import { IMovie, OptionOnSelectData } from '../../shared/types';
 import { TriangleDown12Filled } from '@fluentui/react-icons';
 import { genres } from '../../shared/constants';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+
+const schema = yup
+  .object({
+    title: yup.string().required('Title is required'),
+    vote_average: yup
+      .string()
+      .required('Average vote is required')
+      .matches(/^[1-9]\d*(\.\d+)?$/, 'Invalid average vote, example 7.8'),
+    release_date: yup.string().required('Release Date is required'),
```

Comment:
It should be date. Try to use props `valueAsDate: true` for this field.
OPTIONAL: try to add min and max values for this field also.